src/js/hooks/useTablePagination.js

```diff
@@ -11,6 +11,21 @@ const useTablePagination = ({
   });
 
   const onPageChange = (page) => {
+    const maxPage = Math.floor(totalCount / pagination.pageSize);
+    if (maxPage < page) {
```

Comment:
nitpicky, but since the comparators below start with the `page`, for me personally it would be more readable if it was:
```
if (page > maxPage)
```
but it's my own preference so I'm not requesting a change